BusinessLogic/Services/TimeOffRequestService.cs

```diff
@@ -1,12 +1,61 @@
-﻿using DataAccess.Repository.Interfaces;
+﻿using ApiModels.Models;
+using AutoMapper;
+using DataAccess.Repository;
+using DataAccess.Repository.Interfaces;
+using Domain.EF_Models;
+using Domain.Enums;
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading.Tasks;
 
 namespace BusinessLogic.Services
 {
     public class TimeOffRequestService
     {
+        TimeOffRequestRepository _repository;
+        IMapper _mapper;
+
+        public TimeOffRequestService(TimeOffRequestRepository repository, IMapper mapper)
```

Comment:
register TimeOffRequestRepository as  IRepository<TimeOffRequest, int> and inject IRepository<TimeOffRequest, int> here